parser/gen/CEL.g4

```diff
@@ -173,8 +170,8 @@ NUM_UINT
    ;
 
 STRING
-  : '"' (ESC_SEQ | ~('"'|'\n'|'\r'))* '"'
-  | '\'' (ESC_SEQ | ~('\''|'\n'|'\r'))* '\''
+  : '"' (ESC_SEQ | ~('\\'|'"'|'\n'|'\r'))* '"'
```

Comment:
Use "BACKSLASH" for readability, here and below?
Consider defining DQUOTE and SQUOTE too.

Reply:
I can't unfortunately, I tried. Lexer fragments don't work within the alternative groups.